core/vm/statedb_utils_test.go

```diff
@@ -0,0 +1,125 @@
b60006020820190508181036000830152612ce881612cac565b9050919050565b6000612cfa82611ab0565b9150612d0583611ab0565b9250828201905080821115612d1d57612d1c6127d8565b5b92915050565b60006040820190508181036000830152612d3d81856122f6565b90508181036020830152612d5181846122f6565b90509392505050565b7f455243313135353a207472616e7366657220746f20746865207a65726f20616460008201527f6472657373000000000000000000000000000000000000000000000000000000602082015250565b6000612db6602583611dd2565b9150612dc182612d5a565b604082019050919050565b60006020820190508181036000830152612de581612da9565b9050919050565b7f455243313135353a20696e73756666696369656e742062616c616e636520666f60008201527f72207472616e7366657200000000000000000000000000000000000000000000602082015250565b6000612e48602a83611dd2565b9150612e5382612dec565b604082019050919050565b60006020820190508181036000830152612e7781612e3b565b9050919050565b6000604082019050612e936000830185611b26565b612ea06020830184611b26565b9392505050565b7f455243313135353a2073657474696e6720617070726f76616c2073746174757360008201527f20666f722073656c660000000000000000000000000000000000000000000000602082015250565b6000612f03602983611dd2565b9150612f0e82612ea7565b604082019050919050565b60006020820190508181036000830152612f3281612ef6565b9050919050565b600081519050919050565b600082825260208201905092915050565b6000612f6082612f39565b612f6a8185612f44565b9350612f7a818560208601611de3565b612f8381611c15565b840191505092915050565b600060a082019050612fa360008301886123f9565b612fb060208301876123f9565b8181036040830152612fc281866122f6565b90508181036060830152612fd681856122f6565b90508181036080830152612fea8184612f55565b90509695505050505050565b60008151905061300581611b7c565b92915050565b60006020828403121561302157613020611a48565b5b600061302f84828501612ff6565b91505092915050565b60008160e01c9050919050565b600060033d11156130645760046000803e613061600051613038565b90505b90565b600060443d106130f457613079611a3e565b60043d036004823e80513d602482011167ffffffffffffffff821117156130a15750506130f4565b808201805167ffffffffffffffff8111156130bf57505050506130f4565b80602083010160043d0385018111156130dc5750505050506130f4565b6130eb82602001850186611c55565b82955050505050505b90565b7f455243313135353a207472616e7366657220746f206e6f6e2d4552433131353560008201527f526563656976657220696d706c656d656e746572000000000000000000000000602082015250565b6000613153603483611dd2565b915061315e826130f7565b604082019050919050565b6000602082019050818103600083015261318281613146565b9050919050565b7f455243313135353a204552433131353552656365697665722072656a6563746560008201527f6420746f6b656e73000000000000000000000000000000000000000000000000602082015250565b60006131e5602883611dd2565b91506131f082613189565b604082019050919050565b60006020820190508181036000830152613214816131d8565b9050919050565b600060a08201905061323060008301886123f9565b61323d60208301876123f9565b61324a6040830186611b26565b6132576060830185611b26565b81810360808301526132698184612f55565b9050969550505050505056fea2646970667358221220ffa06f36e5b8b295bd34b370bfbab486876a892926fb6dde98a449f3a880d6aa64736f6c63430008120033`
```

Comment:
I think this is not necessary, we can just use proxy byte code to test for deploying after whitelist deployer v2 hardfork